Services/SystemMenu/main.cpp

```diff
@@ -164,7 +165,10 @@ NonnullRefPtr<GUI::Menu> build_system_menu()
             const auto& bin = g_apps[app_identifier].executable;
             pid_t child_pid;
             const char* argv[] = { bin.characters(), nullptr };
-            if ((errno = posix_spawn(&child_pid, bin.characters(), nullptr, nullptr, const_cast<char**>(argv), environ))) {
+
+            auto af_key = String::formatted("AF_PATH={}", g_apps[app_identifier].af_path);
+            const char* env[] = { af_key.characters(), nullptr };
+            if ((errno = posix_spawn(&child_pid, bin.characters(), nullptr, nullptr, const_cast<char**>(argv), const_cast<char**>(env)))) {
```

Comment:
Is this the only env we pass to our... wait, why is SystemMenu even spawning apps? Shouldn't LaunchServer do that?

Reply:
the env is now handled by `Core::App::Launch()`. SystemMenu and Taskbar both use it. I've been waning to add af files to LaunchServer anyway so that you can put them on your desktop.

Reply:
I've added support for af apps in LaunchServer by default. However, I think that we should not use it to launch apps from SystemMenu, because SystemMenu only ever launches Apps, and I don't see why we would want the behavior of launching an AF app to be user defined.